src/Catalog/Declination.php

```diff
@@ -177,7 +177,7 @@ public function hasVariants(array $variantIds): bool
         $foundIds = 0;
         foreach ($variantIds as $variantId) {
             if (in_array($variantId, $declinationVariantIds)) {
-                $foundIds++;
+                ++$foundIds;
```

Comment:
Ca je ne suis pas fan, faut voir si on désactive la règle

Reply:
Pareil 👎

Reply:
j'avais fait un bench, c'est plus rapide, sur 1M itérations, pas en dessous

Reply:
j'ai viré la règle et revert ces changements